lighthouse-core/audits/uses-rel-preload.js

```diff
@@ -159,7 +159,8 @@ class UsesRelPreloadAudit extends Audit {
     /** @type {Set<string>} */
     const urls = new Set();
     for (const networkRecord of criticalRequests) {
-      if (!networkRecord._isLinkPreload && networkRecord.protocol !== 'data') {
+      if (!networkRecord._isLinkPreload && networkRecord.protocol !== 'data'
```

Comment:
can we move this to an array in url-shim?  

`NON_NETWORK_PROTOCOLS` or something?

There's a few audits where we're checking these individually and i think we can generalize it.